DNAnexus_data_structure.md

```diff
@@ -0,0 +1,129 @@
+# Assembly data structure on DNAnexus
+
+
+## Guidance
+* Each bulletin points are folder names,
+  * with each daughter folders/files shown bellow
+
+```
+File names with extensions are provided in this block.
+```
+
+Once we visit a VGP project, the top level folders are expected to have:
+* genomic_data
+* assembly_\[ver\]
+* transcriptomic_data
+
+## Full sub-folders and files
+
+* genomic_data
+  * pacbio
+    * bam
```

Comment:
We talked briefly with Brett about separating the subreads and scraps BAMs into separate folders to help a bit with selecting inputs to DNAnexus pipelines...

Reply:
Have changed the structure. Let me know what you think.